app/models/reports/schema_v2.rb

```diff
@@ -202,23 +225,34 @@ def sum(region, field)
     end
 
     def summary_field(field)
+      raise ArgumentError, "field #{field} is not part of the FacilityState query" unless field.in?(FIELDS) || field.in?(UNDER_CARE_WITH_LTFU_CALCULATED_FIELDS)
       "sum_#{field}"
     end
 
+    # For some fields we need to sum the under_care numbers with the LTFU numbers to support the "include LTFU"
+    # toggle in reports.
+    def under_care_with_ltfu(field)
+      Arel.sql(<<-EOL)
+        SUM(COALESCE(#{field}_under_care::int, 0) + COALESCE(#{field}_lost_to_follow_up::int, 0)) as sum_#{field}_under_care_with_lost_to_follow_up
+      EOL
+    end
```

Comment:
Ah I see, this is the crux of handling the LTFU toggle. Nice.

Reply:
👍